src/main/java/org/myrobotlab/service/InMoovEyelids.java

```diff
@@ -236,23 +218,19 @@ public boolean save() {
   @Override
   public void startService() {
     super.startService();
-    eyelidleft.startService();
-    eyelidright.startService();
+    eyelidleft.setRest(0.0);

```

Comment:
seems like there should be somewhere else in the life cycle to put this. we shouldn't be adding hard coded values in the startService method here.